README.md

```diff
@@ -37,46 +37,72 @@ There are 2 ways you can set your custom-shell.
 | `zsh`    | Set custom-shell = `zsh`                  |
 | `bash`   | Set custom-shell = `bash`                 |
 | `fish`   | Set custom-shell = `fish`  <°))><         |
+| `nu`     | Set custom-shell = `nu`                   |
 | `ksh`    | Set custom-shell = `ksh` or Kornshell     |
 
 Similarly you can input the name of the shell you want to use.
 <br>
 These commands uses the below command to run the shells-
 
 ```bash
-custom_shell -ic "command";exit
+custom_shell -i -c "command";exit
 ```
 
-For shell's like `nushell` where `-ic` is not supported, you will have to change the args in the `init.lua` file to support it.
-
 ## Keymapping
 
 Add this to your `keymap.toml` file:
 
 To use the `auto` mode, you can set the keymappings as-
 
+```toml
+[[manager.prepend_keymap]]
+on = [ ";" ]
+run = "plugin custom-shell"
+desc = "custom-shell using default shell"
+```
+
+To use the `block` mode, you can set the keymappings as-
+
 ```toml
 [[manager.prepend_keymap]]
 on = [ ":" ]
-run = "plugin custom-shell --args=auto"
-desc = "custom-shell as default"
+run = "plugin custom-shell --args='auto --block'"
+desc = "custom-shell with blocking"
 ```
 
 To choose a specific shell, you can set the keymappings as-
 
 ```toml
 [[manager.prepend_keymap]]
-on = [ ":" ]
-run = "plugin custom-shell --args=zsh" # Example
-desc = "custom-shell as default"
+on = [ ";" ]
+run = "plugin custom-shell --args=zsh"
+desc = "custom-shell with zsh"
+```
+
+To type a command, you can set the keymappings as-
+
+```toml
+[[manager.prepend_keymap]]
+on = [ ";" ]
+run = "plugin custom-shell --args='nu \"ls | sort-by \" --block'"
+desc = "Run ls table in nu sort by ...?"
+```
+
+To run a specific command in a specific shell, you can set the keymappings as-
+
+```toml
+[[manager.prepend_keymap]]
+on = [ ";" ]
+run = "plugin custom-shell --args='fish \"echo example command with --block and --confirm flags ; read c\" --block --confirm'"
+desc = "Run a blocking echo command with fish"
```

Comment:
I dont like the idea to assign `;` with fixed commands as keybindings. They should input the command. But I have alternative for this. Ill tell

Reply:
These are examples, I doubt anyone will use this keybind.